core/src/test/java/io/questdb/test/griffin/AsOfJoinFuzzTest.java

```diff
@@ -102,14 +109,81 @@ private void assertResultSetsMatch() throws Exception {
                 throw new IllegalArgumentException("Unexpected join type: " + joinType);
         }
 
+        StringSink timeFilter = new StringSink();
+        if (exerciseIntervals) {
+            int n = rnd.nextInt(5) + 1;
+            long baseTs = TimestampFormatUtils.parseTimestamp("2000-01-01T00:00:00.000Z");
+            for (int i = 0; i < n; i++) {
+                if (i == 0) {
+                    timeFilter.put("where ts != '");
```

Comment:
`in` or `between` might be a better choice than `ts != <random_value>`. The latter has high chances to appear to be a no-op.